Lesson13/batikan/script.js

```diff
@@ -0,0 +1,178 @@
+
+// 7. Concatenate "Hello" and "World" with a space in between and log the result.
+// CODE HERE
+
+// 8. Check the type of a variable, e.g., let x = 42. Log the type using typeof.
+// CODE HERE
+const x = 42;
+console.log('Ex.8', typeof x);
+// 9. Convert the number 100 to a string and log the result.
+const number = 100;
+console.log('Ex.9', number, number.toString());
+
+// 10. Convert the string "50" to a number and log its type to confirm the conversion.
+console.log('Ex.10', parseInt("50"), parseInt("2.5"), parseInt("2,5"), parseInt("Batikan") );
+
+// 11. Generate a random integer between 2 and 10 and log it.
+console.log(Math.random()); //0.8763250588907179
+const randomFrom2To10 = Math.random() * 8 + 2;
+console.log('Ex.11', Math.floor(randomFrom2To10));
+
+// 12. Round the number 3.7 down using Math.floor and 3.2 up using Math.ceil, log both.
+let xr = Math.floor(3.7);
+console.log(xr);
+let xu = Math.ceil(3.2);
+console.log('Ex.12', xu);
+
+
+// 13. Declare a boolean variable isStudent = true. Log it.
+const isStudent = true;
+console.log('Ex.13', isStudent);
+
+// 14. Initialize counter = 0, then increment it by 1 using counter++ and log it.
+let counter = 0;
+counter ++;
+console.log('Ex.14', counter);
+
+// 15. Initialize points = 10, add 5 to it using points += 5, then log points.
+let points = 10;
+points += 5;
+console.log('Ex.15', points);
+
+// 16. Declare name="Alice", age=30, city="Paris". Log "Alice (30) lives in Paris" using template literals.
+const nName = "Alice"
+let age = 30;
+const city = "Paris"
+console.log('Ex.16', ` ${nName} (${age}) lives in ${city}`);
+
+// 17. Declare variables x=5, y=10, z=15. Log their total sum.
+let xX = 5;
+let yY = 10;
+let zZ= 15;
+console.log('Ex.17', xX + yY + zZ);
+
+// 18. Declare dividend=10 and divisor=3. Log the quotient (divisionResult) and difference (differenceResult).
+let dividend = 10;
+let divisor = 3;
+const differenceResult = dividend / divisor;
+console.log('Ex.18', differenceResult);
+
+// 19. Declare firstName and lastName. Create fullName by concatenating them with a space and log it.
+const firstName = "Batikan";
+const lastName = "Sevil";
+const fullName = firstName + " " + lastName;
+console.log('Ex.19', fullName);
+
+// 20. Declare firstFactor=7 and secondFactor=2. Log the product.
+const firstFactor = 7;
+const secondFactor = 2;
+const product = firstFactor * secondFactor;
+console.log('Ex.20', product);
+
+// 21. Log the value of Math.PI.
+const xPi = Math.PI;
+console.log('Ex.21', xPi);
+
+// 22. Declare counter=0. Increment it using three different methods (e.g., counter++, counter+=1, counter=counter+1) and log the result each time.
+let counterR = 0;
+counterR ++;
+console.log('Ex.22', counterR);
+counterR +=1;
+console.log('Ex.22', counterR);
+counterR = counter+1;
+console.log('Ex.22', counterR);
+
+// 23. Declare initialTemperature=20. Increase it by 5 and log the result.
+let initialTemperature = 20;
+let increasedTemperature = initialTemperature + 5;
```

Comment:
You don't need to use 'let' keyword in this step since we only need to change the value of the existing variable.
```suggestion
increasedTemperature = initialTemperature + 5;
```